As a developer, I can create Gin project from command (#12)
nimblehq/gin-templates: README.md

## Get Started

### Prerequisite 
- [Python interpreter](https://docs.python.org/3/using/index.html)
- Cookiecutter
  - Mac: `brew install cookiecutter`
  - Ubuntu: `brew install cookiecutter`
- Go version >= 1.15

## Usage
- Download **latest** version of gin-templates.
  ```
  go get github.com/nimblehq/gin-templates
  ```

- Run create the project command.
  ```
  gin-templates create
  ```

> - Follow the instruction and the app should be created.

Reviewer: Line break needed and rephrasing needed:
```suggestion
- Follow the instructions in the terminal.
- You new application is created 🎉 .

```

Author: Fixed in abd9be910833e2fd834ea6f9a0349a1cbbd4b5ba
